src/storage/hnsw.cpp

```diff
@@ -189,6 +413,30 @@ int hnsw_search_element (int hnsw_id, DB_VALUE *key_dbvalue, int k, OID *rec_oid
   return NO_ERROR;
 }
 
+static void get_new_hnsw_index_id ()
+{
+  while (true)
+    {
+      if (is_hnsw_index_file_exists (hnsw_index_id))
+	{
+	  hnsw_index_id++;
+	  continue;
+	}
+      else
+	{
+	  if (hnsw_index_map.find (hnsw_index_id) == hnsw_index_map.end())
+	    {
+	      break;
+	    }
+	  else
+	    {
+	      hnsw_index_id++;
+	      continue;
+	    }
+	}
+    }
```

Comment:
1. id를 리턴하는게 getter 함수 특성상 가독성이 좋을 것 같습니다.
2. 다음 처럼 불필요한 continue 조건을 없어고, 조건 검사를 합치면 불필요한 코드 라인을 줄일 수 있습니다. 추가적으로, 필요에 따라 hnsw_index_map.find (hnsw_index_id) != hnsw_index_map.end() 조건도 is_hnsw_index_registered() 정도의 inline 함수로 묶어주면 더 가독성이 좋겠습니다.
```suggestion
while (true)
  {
    if (is_hnsw_index_file_exists (hnsw_index_id) ||
        hnsw_index_map.find (hnsw_index_id) != hnsw_index_map.end())
      {
        hnsw_index_id++;
      }
    else
      {
        break;
      }
  }
```